content/docs/start/experiments.md

```diff
@@ -43,71 +42,87 @@ evaluate:
         cache: false
 ```
 
-The biggest difference is that we see two new sections `metrics` and `plots`
-that are used to mark certain files containing experiment "telemetry". Metric
-files contain scalars (e.g. `AUC`) and plots contain matrices and arrays of
-numbers that are supposed to be visualized to compare or make sense of them.
+The biggest difference to previous stages in our pipeline is in two new
+sections: `metrics` and `plots`. These are used to mark certain files containing
+experiment "telemetry". Metrics files contain simple numeric values (e.g. `AUC`)
+and plots files contain matrices and data series (e.g. `ROC` or model loss
+plots) that meant to be visualizing and compared.
 
-`cache: false` means that those file are small enough and versioned directly
-with Git.
+> `cache: false` means that those file are small enough and versioned directly
+> with Git.
 
 </details>
 
-DVC expects `evaluate.py` to
-[write](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L35)
-model
-[AUC](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5)
-value to the `scores.json` file which is marked as a metrics file with `-M`:
+[`evaluate.py`](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py)
+writes the model's
+[AUC value](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5)
+to `scores.json`, which is marked as a metrics file with `-M`:
 
 ```json
 { "auc": 0.57313829 }
 ```
 
-And it
-[writes](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L31)
-`precision`, `recall`, `thresholds` arrays it gets with the
-[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html)
-call into the `prc.json`:
+It also writes `precision`, `recall`, and `thresholds` arrays (obtained using
+[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
+into plots file `prc.json`:
 
 ```json
 {
   "prc": [
     { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...{ "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
+    ...
+    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
```

Comment:
The formatter wants this to be broken though. Since `...` isn't valid JSON. Comments are also not supported.

Reply:
try to put `'...',`?

Reply:
That works, strangely. Still invalid JSON I would think. Oh well, thanks.